res/wrap/nvml.toml

```diff
@@ -13,6 +13,20 @@ output_ignorelist = [
     "^nvmlC2cModeInfo_v1$",
     "^nvmlGpuInstanceProfileInfo_v3$",
     "^nvmlComputeInstanceProfileInfo_v3$",
+    "nvmlPciInfoExt_v1",
+    "nvmlVgpuHeterogeneousMode_v1",
+    "nvmlVgpuPlacementId_v1",
+    "nvmlVgpuPlacementList_v1",
+    "nvmlVgpuInstancesUtilizationInfo_v1",
+    "nvmlVgpuProcessesUtilizationInfo_v1",
+    "nvmlProcessesUtilizationInfo_v1",
+    "nvmlEccSramErrorStatus_v1",
+    "nvmlSystemConfComputeSettings_v1",
+    "nvmlConfComputeSetKeyRotationThresholdInfo_v1",
+    "nvmlConfComputeGetKeyRotationThresholdInfo_v1",
+    "nvmlGpuFabricInfo_v2",
+    "NVML_DEVICE_PCI_BUS_ID_LEGACY_FMT",
+    "NVML_DEVICE_PCI_BUS_ID_FMT",
```

Comment:
What was the reason to exclude these, all because of `NVML_STRUCT_VERSION`? If something else, please add a small comment.